src/pages/index.js

```diff
@@ -1,35 +1,45 @@
+import Api from '../components/Api.js';
 import Card from '../components/Card.js';
 import FormValidator from '../components/FormValidator.js';
```

Comment:
```js

			this._toggleButtonState();
			evt.target.reset();
``


Please, delete them from lines 84 and 85 from `FormValidator.js`


You need to **reset** the form only  in block `then` in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.